docs/Project-Configuration/Project-Commands.md

```diff
@@ -768,6 +768,188 @@ Parameters:
+                "value": 6.78910,
+                "user_submitted": false
+            },
+            {
+                "name": "dolor",
+                "type": "MIN",
+                "value": 1.11213,
+                "user_submitted": false
+            }
+        ],
+        "sub_tests": []
+    },
+    {
+        "info": {
+            "test_name": "bar",
+            "trial": 0,
+            "tags": [],
+            "args": {}
+        },
+        "created_at": "2023-01-01T09:50:01.720954+00:00",
+        "completed_at": "2023-01-01T10:04:37.015088+00:00",
+        "artifacts": [],
+        "metrics": [],
+        "sub_tests": [
+            {
+                "info": {
+                    "test_name": "WaldoActor.QuxOperation",
+                    "trial": 0,
+                    "tags": [],
+                    "args": {}
+                },
+                "created_at": "2023-01-01T09:51:15.021000+00:00",
+                "completed_at": "2023-01-01T09:51:15.021000+00:00",
+                "artifacts": [
+                    {
+                        "bucket": "genny-metrics",
+                        "path": "WaldoActor.QuxOperation",
+                        "prefix": "foobar_variant.2022_11_bar_patch_ghijk67890_23_01_01_06_24_48_0",
+                        "tags": [],
+                        "created_at": "2023-01-01T09:50:04.992453+00:00",
+                        "local_path": "/data/mci/12345abcdef/build/WorkloadOutput/reports/bar/CedarMetrics/WaldoActor.QuxOperation.ftdc",
+                        "permissions": "public-read",
+                        "convert_bson_to_ftdc": false
+                    }
+                ],
+                "metrics": [],
+                "sub_tests": []
+            },
+            {
+                "info": {
+                    "test_name": "FredActor.BazOperation",
+                    "trial": 0,
+                    "tags": [],
+                    "args": {}
+                },
+                "created_at": "2023-01-01T09:51:11.175000+00:00",
+                "completed_at": "2023-01-01T09:51:14.552000+00:00",
+                "artifacts": [
+                    {
+                        "bucket": "genny-metrics",
+                        "path": "FredActor.BazOperation",
+                        "prefix": "foobar_variant.2022_11_bar_patch_ghijk67890_23_01_01_06_24_48_0",
+                        "tags": [],
+                        "created_at": "2023-01-01T09:50:04.992453+00:00",
+                        "local_path": "/data/mci/12345abcdef/build/WorkloadOutput/reports/bar/CedarMetrics/FredActor.BazOperation.ftdc",
+                        "permissions": "public-read",
+                        "convert_bson_to_ftdc": false
+                    }
+                ],
+                "metrics": [],
+                "sub_tests": []
+            },
+        ]
+    }
+]
+```
+
+Each test result object holds data about a specific test and its
```

Comment:
Could you organize this by defining each object with a table, similar how you did for the `Artifact` object—[here](https://github.com/evergreen-ci/cedar/wiki/Rest-V1-Usage#objects-1) is an example of how I did it in the Cedar documentation. I think it will be more readable that way.

Reply:
Good idea, thank you for the example!